nwb_healthstatus/__main__.py

```diff
@@ -52,7 +52,7 @@ def test(casefile, samples_path):
             with pynwb.NWBHDF5IO(str(filepath), mode='r') as io:
                 ## TODO: Capture and display possible warnings
                 obj = io.read()
-            case.test(obj)
+                case.test(obj)
```

Comment:
was there a need for this? IIRC `.read` should read it entirely and thus it should be ok to keep it outside of the context handler.  Or is that not the case?

Reply:
Yes, otherwise I can't make the test pass since I can't access the data inside the NWB file, I get: `<Closed HDF5 dataset>`.